api/src/utilities/model-fields.ts

```diff
@@ -1,6 +1,6 @@
 import { Prisma } from '@prisma/client';
 
-export const getModelFields = (modelName: string) => {
+export const getModelFields = (modelName: string): any => {
```

Comment:
This was to fix a ts issue. Its not ideal but was the only way to make the swagger command work. Please let me know if you know a better way.

Reply:
It was getting this:

```
 yarn generate:client
yarn run v1.22.22
$ ts-node scripts/generate-axios-client.ts && prettier -w ../shared-helpers/src/types/backend-swagger.ts
loaded spec document fail! http://localhost:3100/api-json
../shared-helpers/src/types/backend-swagger.ts 606ms
Done in 1.77s.
```

Then I saw those files were throwing an error. When I add any then the command passed.  But I Can revert that and retry or use that type